### PR TITLE
SLI-1551 Fix AI Fix Suggestion Navigation

### PR DESCRIPTION
**Pull request by eray-felek-sonarsource**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt [hunk 1]
```diff
@@ -37,9 +38,10 @@ import org.sonarsource.sonarlint.core.rpc.protocol.client.fix.FixSuggestionDto
 class ShowFixSuggestion(private val project: Project, private val file: VirtualFile, private val fixSuggestion: FixSuggestionDto) {
 
     fun show() {
+        var hasNavegated = false
```
Comment (thahnen) [inline src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt:41]: `hasNavigated` but I don't care for a typo, only if you fix the other typo as well.

File: src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt [hunk 2]
```diff
@@ -55,10 +57,23 @@ class ShowFixSuggestion(private val project: Project, private val file: VirtualF
             var successfullyOpened = true
 
             fixSuggestion.fileEdit().changes().forEachIndexed { index, change ->
-                fileEditorManager.selectedTextEditor?.let {
-                    val startLine = change.beforeLineRange().startLine
-                    val endLine = change.beforeLineRange().endLine
+                val startLine = change.beforeLineRange().startLine
+                val endLine = change.beforeLineRange().endLine
+
+                if(!hasNavegated) {
```
Comment (thahnen) [inline src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt:63]: Add a space between the `if` and its brackets. Only if you fix the other typo as well.

**Review by thahnen — APPROVED** [review]
LGTM, the fixes proposed are optional.

File: src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt (hunk 2 above)
Comment (eray-felek-sonarsource) [reply src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt:63]: Since there are 2 more bugs to be fixed, I will change them while fixing those

**Pull request merged**